<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmEstimationsAdd</class>
 <widget class="QDialog" name="frmEstimationsAdd">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New estimation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/dinero.png</normaloff>:/images/dinero.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 192, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Add a year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtYear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblEstimation">
         <property name="text">
          <string>Add a estimation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQLineEdit" name="txtDPA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Add a source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSource"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Save estimation</string>
       </property>
       <property name="icon">
        <iconset resource="../images/xulpymoney.qrc">
         <normaloff>:/xulpymoney/save.png</normaloff>:/xulpymoney/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtDPA</tabstop>
  <tabstop>txtYear</tabstop>
  <tabstop>cmbSource</tabstop>
  <tabstop>cmd</tabstop>
 </tabstops>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
